etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml

```diff
@@ -13,8 +13,25 @@ instances:
   name: RabbitMQ
   nodes:
   - "rabbit@{{ inventory_hostname }}"
-  queues:
-  - conductor
```

Comment:
No more conductor?

Reply:
Do we need it @dougszumski ?

Reply:
I think it came from the default plugin config. For the Rabbit dashboard that we have, we focus on unacknowledged messages and message rates, which include the conductor (I assume it is Nova conductor). This PR matches the config we have for a working dashboard in another deployment. I think it's good to go.